core/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -24,6 +24,9 @@
 import org.elasticsearch.ElasticsearchException;
 import org.elasticsearch.ExceptionsHelper;
 import org.elasticsearch.action.ActionRequest;
```

Comment:
we can remove the unused imports here (namely: `ActionRequest`, `DocWriteRequest.OpType`, `ReplicationRequest`, `ShardId` and `RestStatus`)